core/trino-grammar/src/main/antlr4/io/trino/grammar/sql/SqlBase.g4

```diff
@@ -155,6 +162,7 @@ statement
     | EXPLAIN ANALYZE VERBOSE? statement                               #explainAnalyze
     | SHOW CREATE TABLE qualifiedName                                  #showCreateTable
     | SHOW CREATE SCHEMA qualifiedName                                 #showCreateSchema
+    | SHOW CREATE CATALOG identifier                                   #showCreateCatalog
```

Comment:
The current implementation exposes all properties even if it's marked as `@ConfigHidden` or `@ConfigSecuritySensitive`. I think we should hide or redact such properties.

Reply:
@ssheikin could you maybe extract `SHOW CREATE CATALOG` to a separate PR? @martint would that help with getting the less controversial parts merged sooner?